built_value_generator/test/built_value_generator_test.dart

```diff
@@ -745,6 +745,21 @@ abstract class Value implements Built<Value, ValueBuilder> {
         contains(
             '1. Set `nestedBuilders: false` in order to use `comparableBuilders: true`.'));
   });
+
+  test('Cleans generated class names for private classes', () async {
```

Comment:
Please also add a private class to `end_to_end_test/lib/values.dart`.

Reply:
Done!